.github/workflows/github-ci.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
```

Comment:
We should perhaps also make sure we're ignoring `sample-files` from flake8 if it's not already? Would be annoying to be blocked on updating this repository for something up with that one's python code styling.

Reply:
:+1:

Reply:
Very good point! Thanks for raising it!